src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/document-[document]/attributes/enum.svelte

```diff
@@ -7,13 +7,20 @@
     export let value: string;
     export let attribute: Models.AttributeEnum;
     export let optionalText: string | undefined = undefined;
+    export let disabled = false;
 
-    $: options = attribute.elements.map((element) => {
-        return {
-            label: element,
-            value: element
-        };
-    });
+    $: options = [
+        ...attribute.elements.map((element) => {
+            return {
+                label: element,
+                value: element
+            };
+        }),
+        !attribute.required && {
+            label: 'NULL',
+            value: null
+        }
+    ].filter(Boolean);
```

Comment:
Removing falsy values. In case attribute is required, this snippet will return false:
```
!attribute.required && {
            label: 'NULL',
            value: null
        }
```

And we don't want to pass in `false` as an option

Reply:
I'm more focused on the `.filter(Boolean)` part. It seems like that always returns true:

<img width="362" alt="image" src="https://user-images.githubusercontent.com/1477010/231000493-0db41500-1d1a-4617-9180-3896e4468e62.png">